liferay-echo/liferay-portal
# LPS-200901 Visual changes in segment editor to properties menu

Motivation
=======
[Link to story](https://liferay.atlassian.net/browse/LPS-200901)

Steps to Verify:
----------------
1. Go to People -> Segments
1. Click on Add new user segment
1. Check that the visual changes have been applied correctly

Screenshots (optional):
-----------------------


![Captura de pantalla 2023-11-06 a las 16 37 54](https://github.com/liferay-echo/liferay-portal/assets/3276218/37a076fb-78fd-4dde-82c4-46a5ccf7b2df)
